.github/scripts/validate-docs-links.js

```diff
@@ -22,21 +30,26 @@ const validateUrl = async (kind, name, documentationUrl) =>
 					port: 443,
 					path: url.pathname,
 					method: 'HEAD',
+					agent,
+				},
+				(res) => {
+					debug('\u2713', kind, name);
```

Comment:
What is this magic string? Could be moved to a constant with some descriptive name

Reply:
Could also use the character directly: ✓

Reply:
I still haven't gotten used to putting unicode chars or emojis directly in code. 
I guess I'm showing signs of being old.
![image](https://github.com/n8n-io/n8n/assets/196144/cd647e93-8c4f-4a14-98d2-a09fd54d6c3e)